### ISSUE TITLE
Provide es module version

### ISSUE THREAD

**Issue by stefnotch** [GregRos/parjs#39]
I'm struggling with importing your library in a `vite` project. I mostly just get the error

```
Uncaught SyntaxError: import not found: default
```

**Comment by davidcallanan** [comment]
I agree, an ES module version would be nice. I am struggling to import it ATM.

It's a matter of adding `"module": "index.esm.js"` to the `package.json` and then including an ESM building step.

**Comment by GregRos** [comment]
Maintaining multiple builds is actually way more frustrating than it sounds! There is a reason why libraries started dropping CommonJS support a few years ago, even before TypeScript properly supported it.

My current plan is to move the library to ESM as part of a major upgrade  sometime in the future.

I'll try to wrangle multiple builds if it gets more support though! 😊

**Comment by stefnotch** [comment]
@GregRos That sounds like a lovely plan, and I fully support it.
